examples/typescript-backbone/js/app.ts

```diff
@@ -339,6 +396,20 @@ class AppView extends Backbone.View {
 		Todos.each(this.addOne);
 	}
 
+	// Filter out completed/remaining tasks
+	filter(criteria: string) {
+		taskFilter = criteria;
```

Comment:
> If the TodoMVC app is this much of a mess I would not like to see how a "real" app came out!

Such a great point. There are a lot of complaints about TodoMVC not being complex enough to reflect the requirements of real-world applications, but the test definitely works in this direction ...